control-plane/src/modules/runs/agent/tools/cluster-internal-tools.ts

```diff
@@ -40,7 +41,9 @@ export const getClusterInternalTools = async (
     tools[ACCESS_KNOWLEDGE_ARTIFACTS_TOOL_NAME] = buildAccessKnowledgeArtifacts;
```

Comment:
`ACCESS_KNOWLEDGE_ARTIFACTS_TOOL_NAME` should technically be included in std library also